static/networks/nerpa.json

```
@@ -51,8 +51,9 @@
   ],
   "Git":{
     "Base": "https://github.com/filecoin-project/lotus/",
-    "Commit":"b55ca92d339f3eeb09d5b6e059e82ee957156f7e",
-    "Branch":"release/v1.5.3"
+    "Commit":"3e6dcb578b1a6d7b5ec60cfe3e37601a44fe071a",
+    "Branch":"master",
+    "Tag":"v1.5.3"
```

@eshon branch name is inaccurate ( our new branching regime for lotus is a bit confusing )
```suggestion
    "Commit":"3e6dcb578b1a6d7b5ec60cfe3e37601a44fe071a",
    "Branch":"release/v1.5.3",
    "Tag":"v1.5.3"
```

---

The UX is confusing, that means "this commit was merged into branch master, and is now contained in it at some point"
```
~/devel/lotus$ git rev-parse origin/release/v1.5.3
3e6dcb578b1a6d7b5ec60cfe3e37601a44fe071a

~/devel/lotus$ git rev-parse v1.5.3
3e6dcb578b1a6d7b5ec60cfe3e37601a44fe071a
```

---

If it's contained in master, do we still need to ask users switch to the branch `release/v1.5.3` to download the tag?

---

What is our current recipe? ( what do we tell users to do )

---

Just updated the [Docs](https://docs.filecoin.io/get-started/lotus/installation/#build-and-install-lotus):
- For mainnet, it's master + latest release
- For other networks, we tell them:

> For networks other than mainnet, look up the current branch or tag and commit for the network you want to join in the [Filecoin networks dashboard](https://networks.filecoin.io/), then build Lotus for your specific network below.
```
git checkout <branch_or_tag>
# For example:
git checkout master # mainnet
git checkout <vX.X.X> # tag for latest release
```

It's a little confusing, so if they don't have to switch branches they

---

Right: both `checkout master` and `checkout vX.X.X` will build them a nerpa
The difference is that with `vX.X.X` they will get the `latest stable release`
If they build from master they will get ( currently ) `v1.6.X-prealpha`. While master generally *should* be relatively stable, it also contains quite a hefty amount of new, potentially undertested code. So things might not-exactly-work.

TLDR:
- If you are integrating/testing against "official lotus" - you `git checkout v1.5.3` ( or later when we have later )
- If you are testing some just-released-feature, preparing for an upcoming API change, etc etc etc - you use the just-freh-from-the-oven `master`

@arajasek ^^ can you confirm I described our current branching process correctly?

---

@ribasushi I'm just going to remove the branch so we can close this.